src/invariants/token.ts

```diff
@@ -1,7 +1,24 @@
 import { BigNumber } from "bignumber.js";
 import { ERC20Contract } from "../wrappers";
 
+import * as singleLineString from "single-line-string";
+
+export const TokenAssertionErrors = {
+    MISSING_ERC20_METHOD: () => singleLineString`Contract does not implement ERC20 interface.`,
```

Comment:
Let's add an `address` parameter and specify _which_ contract does not implement the required interface. 

i.e. `Contract at address ${contractAddress} does not implement ERC20 interface `